src/Microsoft.Data.SqlClient/netfx/src/Microsoft/Data/SqlClient/TdsParserStateObject.cs

```diff
@@ -35,12 +50,6 @@ internal partial class TdsParserStateObject
         // Timeout variables
         internal bool _attentionReceived = false;               // NOTE: Received is not volatile as it is only ever accessed\modified by TryRun its callees (i.e. single threaded access)
```

Comment:
Could you make private this field and add the same property with get/set on it  as NetCore:
```csharp
        internal bool HasReceivedAttention
        {
            get => _attentionReceived;
            set => _attentionReceived= value;
        }
```